app/javascript/components/mappers/domainToApi/lease.js

```diff
@@ -1,5 +1,4 @@
 import { createFieldMapper } from '~/utils/objectUtils'
-import { currencyToFloat } from '~/utils/utils'
```

Comment:
is this still in use anywhere? If not can we delete it?

Reply:
removed and removed test